src/RestControllers/AuthorizationController.php

```diff
@@ -655,6 +577,7 @@ public function getAuthorizationServer(): AuthorizationServer
             $responseType->markIsAuthorizationGrant(); // we have specific SMART responses for an authorization grant.
         }
 
+        $this->logger->debug("key log ", ['privateKey' => $this->privateKey, 'passphrase' => $this->passphrase, 'encryptionKey' => $this->oaEncryptionKey]);
```

Comment:
I prob wouldn't log these actual values in a log file (since would consider them very sensitive info from security perspective). Since a novice user could have debug on at some point and potentially pass their log somewhere in unsecure fashion.

Reply:
maybe just show if they are empty or not if helpful for debug purposes

Reply:
Wow... that was only for my own debugging purposes.  Can't believe I left that in there.  I'm really, really glad you caught that.